### ISSUE TITLE
It would be great if PsMake used PowerShell Nuget commands instead of Nuget.exe 

### ISSUE THREAD

**Issue by geoffclark** [Suremaker/PsMake#10]
It would be great to be able to use PsMake with PowerShell .NetCore version.

**Comment by Suremaker** [comment]
I started looking how to implement the ps5 functions instead of nuget.exe: https://github.com/Suremaker/PsMake/tree/oneget

... but there are issues with it:
* powershell version does not support nuget v3 api (only v2),
* I did not found a way to configure the Find-Package / Install-Package to use nuget.config file to configure the repositories. It is bad, because if the repository sources would not be controlled locally, there would be some conflicts happening in future..

I have to play a bit more with it to sort it out - but feel free to take a look at that branch and maybe play with it.

Basically, the psmake got a new parameter -pp accepting the package providing method: nugetexe or powershell
